### DISCUSSION TITLE
Restore minimized RDP window when the user tried to start an already running RDP connection

### DISCUSSION THREAD

**Discussion by MWinhard** [1Remote/1Remote#381] [Ideas]
Sometimes an RDP connection is already running but the PRemoteM host window is minimized. I choose the same RDP connection from the system tray icon's context menu. The chosen RDP connection is made the active connection within the PRemoteM host window.

However, the PRemoteM host window is still minimized. How about restoring the PRemoteM host window?

The desired behavior already exists when the PRemoteM host window is minimized but the choesen RDP connection is _not_ already running.

**Comment by majkinetor** [comment]
https://github.com/1Remote/1Remote/issues/291

It indeed doesn't work, I will reopen the ticket.

**Comment by MWinhard** [comment]
Thank you for the fix. Will this be part of the next regular release or will you offer a bugfix release?

**Reply by VShawn** [reply]
i won't publish it right now. but you can get the fix from github action artifact on dev branch. 

sorry for i cannot give you
 a link now,  i can't get the link within my mobile phone.

**Comment by MWinhard** [comment]
No problem. I can wait for the next official release.

**Comment by MWinhard** [comment]
Couln't resist. 😉

I installed the latest .NET 6 action artifact in my VM at the company office. I can confirm that the fix works as expected.

The installation on my physical pc at home stays at the official release (currently 0.7.2.7).
